hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/INodeAttributeProvider.java

```diff
@@ -405,6 +405,30 @@ default void checkPermissionWithContext(AuthorizationContext authzContext)
           + "implement the checkPermissionWithContext(AuthorizationContext) "
           + "API.");
     }
+
+    /**
+     * Checks if the user belongs to superuser group.
```

Comment:
Can we rephrase "Checks if the user belongs to superuser group." to "Checks if the user is a superuser or belongs to superuser group."